### ISSUE TITLE
Removing extra bottom margin in Admonition blocks inside tables

### ISSUE THREAD

**Issue by PauloFrancaLacerda** [asciidoctor/asciidoctor#2568]
Sorry for the long topic title. Please, feel free to fix it as needed.

Well, I noticed there was something (visually) uncomfortable in my admonition blocks within tables. I'm not a webdesigner, so it took me some time to find the bottom margin in admonition's inner table.

After struggling for a while, I managed to do a hard-and-dirty hack to the CSS, for that margin to be removed selectively, wherever necessary. In such cases, now I add a custom role to the table that embraces the admonition block:

`[cols="<a", caption="", options="autowidth", role="NoAdmonitionBottomMargin"]`

The CSS hack:
`table.NoAdmonitionBottomMargin div.admonitionblock table {margin-bottom:0}`

Screenshot (hacking the default css):
http://blaisepoint.online/img/AdmonInTables-1.png

Screenshot (hacking my custom css):
http://blaisepoint.online/img/AdmonInTables-2.png

The test source file:
http://blaisepoint.online/files/AdmonInTables.adoc

**Question**: Is there any easier way of doing this?

**Comment by mojavelinux** [comment]
You are absolutely correct. The bottom margin of the last block in an AsciiDoc table cell is not being collapsed.

When I looked into this issue, I noticed a few things that could be improved:

* add the "content" class to the div that encloses the AsciiDoc cell content for easier styling
* collapse the bottom margin of the last block in an AsciiDoc cell
* simplify the styles used to collapses the bottom margin of the last paragraph in a normal cell
* remove some styles which weren't being used at all

I've submitted these changes in #2593.
